### PR TITLE
Added the aquawave as parameter for visibility

### PR DESCRIPTION
**Pull request by maheshopsmx**

### CODE REVIEW

File: charts/oes/charts/spinnaker/config/orca-local.yml [hunk 1]
```diff
@@ -243,7 +243,7 @@ job:
         waitForCompletion: true
         application: spin
         parameters:
-          - defaultValue: "http://oes-sapor:8085"
+          - defaultValue: "http://oes-sapor.<NAMESPACE OF OES>:8085"
```
Comment (abhinaybyrisetty) [inline charts/oes/charts/spinnaker/config/orca-local.yml:246]: You don't need namespace if the custom stage job is in the same namespace. Who'll replace this before installing?

**Review by abhinaybyrisetty — CHANGES_REQUESTED** [review]

File: charts/oes/charts/spinnaker/config/orca-local.yml (hunk 1 above)
Comment (abhinaybyrisetty) [reply charts/oes/charts/spinnaker/config/orca-local.yml:246]: Perhaps, we can simply keep it as old value as the default. Let's not change it

Comment (maheshopsmx) [reply charts/oes/charts/spinnaker/config/orca-local.yml:246]: ok Sure Abhinay

**Review by abhinaybyrisetty — APPROVED** [review]
Ok

**Pull request merged**